code/modules/assembly/mousetrap.dm

```diff
@@ -114,23 +114,27 @@
 	update_appearance()
 	var/obj/item/bodypart/affecting = null
 	if(ishuman(target))
-		var/mob/living/carbon/human/H = target
-		if(HAS_TRAIT(H, TRAIT_PIERCEIMMUNE))
+		var/mob/living/carbon/human/victim = target
+		if(HAS_TRAIT(victim, TRAIT_PIERCEIMMUNE))
 			playsound(src, 'sound/effects/snap.ogg', 50, TRUE)
 			pulse(FALSE)
 			return FALSE
 		switch(type)
 			if("feet")
-				if(!H.shoes)
-					affecting = H.get_bodypart(pick(BODY_ZONE_L_LEG, BODY_ZONE_R_LEG))
-					H.Paralyze(60)
+				if(!victim.shoes)
+					affecting = victim.get_bodypart(pick(BODY_ZONE_L_LEG, BODY_ZONE_R_LEG))
+					victim.Paralyze(60)
+				else
+					to_chat(victim, span_notice("Your [victim.shoes] protects you from the [src]."))
```

Comment:
```suggestion
					to_chat(victim, span_notice("Your [victim.shoes] protects you from [src]."))
```

`the` is implied. This would've been `the the mouse trap`